examples/pgbouncer/README.md

```diff
@@ -0,0 +1,260 @@
+# Using the Cloud SQL Auth Proxy with PgBouncer
```

Comment:
Does the filepath for this make sense? Should we have both of these under the k8s folder? Maybe one folder for "sidecar" and one for "service"?

Reply:
That's a good idea.